### ISSUE TITLE
Cannot log into HOOBS.local despite previous install

### ISSUE THREAD

**Issue by gulyassw** [hoobs-org/HOOBS#1686]
**Description**
Cannot access HOOBS.local in browser (Safari)

**Version**
HOOBS 3.3.?

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [ X] Yes
- [ ] No

**Pervious version**
I don't recall exactly but original install was definitely not older than 3.3.2

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [ x] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [ ] HOOBS SD Card
- [ X] Custom Install

**If this is a custom install, what's your operating system?
MacOS Big Sur 11.6.1

**List your plugins**
Ring

**Post your config**

```
➜  ~ cat ~/.hoobs/etc/config.json
{
    "server": {
        "port": 80,
        "origin": "*",
        "autostart": 0,
        "home_setup_id": "X-HM://XXXXXXXXXXXXX”,
        "polling_seconds": 5
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "tealzier",
        "locale": "en",
        "temp_units": "fahrenheit",
        "country_code": "US",
        "postal_code": “XXXXX”
    },
    "bridge": {
        "name": "HOOBS",
        "port": XXXXX,
        "pin": “XXX-YY-ZZZ”,
        "username": “XX:XX:XX:XX:XX:XX”
    },
    "description": "",
    "ports": {},
    "plugins": [],
    "accessories": [],
    "platforms": [],
    "package_manager": "npm",
    "system": "hoobs"

```

**Post your log**

```
Cannot access HOOBS screen/log
```

**Additional context**
No hardware being used - running in MacOS Unix only.  Had this working in the past with HomeKit.  Have not paid attention the last few months and did not notice HOOBS was not working.  May have updated MacOS in this timeframe.  Looking to upgrade now to HOOBS 4 and just want to preserve any settings if I can for migration.  If I cannot, then I need instructions to fully nuke the prior install and start afresh.

**Comment by BobbySlope** [comment]
Try to run the fix command:

https://support.hoobs.org/topics/61c9df1fdcf29f5e39f02f34/61c9dfd0dcf29f5e39f030e3

**Comment by gulyassw** [comment]
Hello,Can you please advise if this FIX command was successful or what I may be doing wrong?  I have seen no change in the HOOBS updater after this command was executed.Sent from my iPhoneOn Dec 28, 2021, at 14:20, BobbySlope ***@***.***> wrote:﻿
Closed #1686.
